core/src/main/java/org/mule/processor/AsyncInterceptingMessageProcessor.java

```diff
@@ -72,7 +72,7 @@ public WorkManager getWorkManager() throws MuleException
 
     public void start() throws MuleException
     {
-        if (workManager != null)
+        if (workManager != null && !workManager.isStarted())
```

Comment:
Was the start phase being invoked twice and that is why you added that extra condition?